expression/aggregation/agg_to_pb.go

```diff
@@ -83,6 +87,27 @@ func AggFuncToPBExpr(sc *stmtctx.StatementContext, client kv.Client, aggFunc *Ag
 		}
 		children = append(children, pbArg)
 	}
+	if tp == tipb.ExprType_GroupConcat {
+		orderBy := make([]*tipb.ByItem, 0, len(aggFunc.OrderByItems))
+		for _, arg := range aggFunc.OrderByItems {
+			pbArg := expression.SortByItemToPB(sctx.GetSessionVars().StmtCtx, client, arg.Expr, arg.Desc)
+			if pbArg == nil {
+				return nil
+			}
+			orderBy = append(orderBy, pbArg)
+		}
+		// encode GroupConcatMaxLen
+		GCMaxLen, err := variable.GetSessionOrGlobalSystemVar(sctx.GetSessionVars(), variable.GroupConcatMaxLen)
+		if err != nil {
+			panic(fmt.Sprintf("Error happened when buildGroupConcat: no system variable named '%s'", variable.GroupConcatMaxLen))
+		}
+		maxLen, err := strconv.ParseUint(GCMaxLen, 10, 64)
+		// Should never happen
+		if err != nil {
+			panic(fmt.Sprintf("Error happened when buildGroupConcat: %s", err.Error()))
```

Comment:
It's rough to panic directly. We can simply log warning here and return nil.

Reply:
ok